test/Integration/tpp-add.mlir

```diff
@@ -0,0 +1,87 @@
+// RUN: tpp-opt %s -convert-linalg-to-tpp -convert-linalg-to-loops -convert-tpp-to-loops -convert-vector-to-scf -convert-scf-to-cf -expand-strided-metadata -lower-affine -convert-arith-to-llvm -convert-vector-to-llvm -convert-memref-to-llvm -arith-expand -convert-math-to-llvm -convert-func-to-llvm -canonicalize -reconcile-unrealized-casts | \
+// RUN: mlir-cpu-runner \
```

Comment:
Can you try to use `tpp-run` instead? We have more checks than `mlir-cpu-runner`.

Reply:
Getting: Main function has to be 'void', even if the kernel return's a value, because that's the type of the wrapper we create here

Reply:
`tpp-run` can cope with `void main`. Also, please replace the long list of passes with the default pipeline. This could probably just be a single line.